src/app/ReadClient.cpp

```diff
@@ -720,5 +753,42 @@ CHIP_ERROR ReadClient::SendSubscribeRequest(ReadPrepareParams & aReadPreparePara
     return CHIP_NO_ERROR;
 }
 
-}; // namespace app
-}; // namespace chip
+void ReadClient::OnResubscribeTimerCallback(System::Layer * apSystemLayer, void * apAppState)
+{
+    ReadClient * const client = reinterpret_cast<ReadClient *>(apAppState);
```

Comment:
I usually take the approach of naming it as `_this` to better convey the fact that the appState is just a pointer to ourselves.

Reply:
updated, thanks